addons/rearm/functions/fnc_addVehicleMagazinesToSupply.sqf

```diff
@@ -0,0 +1,49 @@
+/*
+ * Author: GitHawk
+ * Adds all magazines of a vehicle to the supply.
+ *
+ * Arguments:
+ * 0: Ammo Truck <OBJECT>
+ * 1: Vehicle object or Vehicle class <OBJECT><STRING>
```

Comment:
For 2nd arg string is fine
`params [["_truck", objNull, [objNull]], ["_vehicle", objNull, [objNull, ""]]];`